src/main/java/com/bp3/camunda/camunda7/C7RESTConnector.java

```diff
@@ -63,6 +68,19 @@ public void execute(ExternalTask externalTask, ExternalTaskService externalTaskS
         Map<String, String> httpHeaders = (Map<String, String>) getVariable(externalTask, PARAM_HTTP_HEADERS, Map.class);
         Map<String, String> httpQueryParams = (Map<String, String>) getVariable(externalTask, PARAM_HTTP_PARAMETERS, Map.class);
         String outputVariableName = (String) getVariable(externalTask, PARAM_OUTPUT_VARIABLE, String.class);
+        String errorHandlingMethod = (String) getVariable(externalTask, PARAM_ERROR_HANDLING_METHOD, String.class);
+
+        int totalRetries = 0; // Default to zero if not set
```

Comment:
You only need the retries and retryBackoff if the error handing method is "Failure", maybe this code should be moved there.